src/xtopcom/xdata/src/xbstate_ctx.cpp

```diff
@@ -699,4 +718,10 @@ int32_t xbstate_ctx_t::set_tep_balance_bytes(common::xtoken_id_t const token_id,
     return set_tep_balance_bytes(top::to_string(token_id), new_balance);
 }
 
+common::xaccount_address_t xbstate_ctx_t::account_address() const {
```

Comment:
put to construct function?

Reply:
constructor is not a good solution for now, it throws if error occurs. ec makes sure no exception will be thrown.